knit/src/Knit/Printer.hs

```diff
@@ -75,8 +75,14 @@ ppExpr =
 
     ppArg = \case
       ArgPos a -> parensIfProcCall a (ppExpr a)
-      ArgKw name a -> nameToDoc name PP.<> PP.colon PP.<+>
-        parensIfProcCall a (ppExpr a)
+      ArgKw name a -> nameToDoc name PP.<> PP.colon PP.<+> (
```

Comment:
That's a very bad place to add this logic. What if the password is passed as a positional argument?

Reply:
You are right, I will.